### DISCUSSION TITLE
Display Yammer Communities in search results

### DISCUSSION THREAD

**Discussion by zeair** [microsoft-search/pnp-modern-search#2228] [Q&A] [answered]
Hi, 

Is there a way to include yammer communities in search results (whith link to the community instead of site collection)?

**Comment by FranckyC** [comment, marked as answer]
Hi @zeair, unfortunately, I don't think there is a native way to do that. However, you could write your own Microsoft Search connector to crawl Yammer content. More complicated but works. See https://docs.microsoft.com/en-us/graph/api/resources/indexing-api-overview?view=graph-rest-1.0
